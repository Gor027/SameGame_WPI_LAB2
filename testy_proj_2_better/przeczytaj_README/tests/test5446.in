  +II +     + ++   I
II IIII+  +++I+I+
 I   +I I +++ + I+  +     III +I+  +I II+I II+
 +  II+II+ +++ +I+II+ + +  I+ II+  +I+ I  ++I+I++I I  I++   ++I +I+++I+I
I  +  I++I++I + I+I I I+I+ III I +++++   +I I ++I+I+I
 I +I +  III+ +++I + I++I++I  
I  + II II +++II I +I++I  
 II + + +  I    +IIIII I ++ +++ +    + I
