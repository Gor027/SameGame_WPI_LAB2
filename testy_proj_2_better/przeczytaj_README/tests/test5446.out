     +
     I    ++
     I    ++ + ++
  II + I  ++ ++I+ I+
  IIII I++++ +II+ I+ ++   II
I +I++++I+III++III+III+++II+ +I+  +I+ I    I
I +IIIIII+++++++II+I++II++IIIII+  +I+ I I+II+I++I I
++++I+++III++II+IIIII+II++I+++I++++++II++I+I+++I+I+II++++I+I+++I+I
